src/Event/ReadEventRestController.php

```diff
@@ -87,4 +100,42 @@ public function history($cdbid)
 
         return $response;
     }
+
+    /**
+     * @param string $cdbid
+     *
+     * @return string
+     */
+    public function getCalendarSummary($cdbid, Request $request)
+    {
+        $data = null;
+        $response = null;
+
+        $style = ($request->query->get('style') !== null) ? $request->query->get('style') : 'text';
+        $langCode = ($request->query->get('langCode') !== null) ? $request->query->get('langCode') : 'nl_BE';
+        $hidePastDates = ($request->query->get('hidePast') !== null) ? $request->query->get('hidePast') : false;
+        $timeZone = ($request->query->get('timeZone') !== null) ? $request->query->get('timeZone') : 'Europe/Brussels';
+
+        if ($request->query->get('format') !== null) {
```

Comment:
You can increase the readability of your code by reducing the if/else construct to an if, which checks if the format is not there and immediately return the error response.
Or, even simpler, just assume a default value for the format.